cli/ops/deploy.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+use deno_core::op2;
+use deno_error::JsErrorBox;
+
+use crate::tools::deploy::get_token_entry;
+
+deno_core::extension!(
+  deno_deploy,
+  ops = [
+    op_deploy_token_get,
+    op_deploy_token_set,
+    op_deploy_token_delete,
+  ],
+);
+
+#[op2]
+#[string]
+pub fn op_deploy_token_get() -> Result<Option<String>, JsErrorBox> {
+  match get_token_entry()
+    .map_err(|e| JsErrorBox::type_error(e.to_string()))?
+    .get_password()
+  {
+    Ok(password) => Ok(Some(password)),
```

Comment:
Is it ok this is accessible without permissions?

Reply:
these ops are only accessible by the `deploy` subcommand